htdocs/comm/action/class/actioncomm.class.php

```diff
@@ -1336,7 +1336,7 @@ public function getNomUrl($withpicto = 0, $maxlength = 0, $classname = '', $opti
 			if ($add_save_lastsearch_values) $url.='&save_lastsearch_values=1';
 		}
 
-		$linkstart = '<a href="'.$url.'"';
+		$linkstart = '<a href="'.$url.'" title="'.$label.'"';
```

Comment:
The tag title is already included into $linkclose so a full tooltip with complete data are shown